parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -82,7 +102,7 @@ $global:SubscriptionId = "{{GetVariable "subscriptionId"}}"
 $global:ResourceGroup = "{{GetVariable "resourceGroup"}}"
 $global:VmType = "{{GetVariable "vmType"}}"
 $global:SubnetName = "{{GetVariable "subnetName"}}"
-$global:MasterSubnet = "{{GetWindowsMasterSubnetARMParam}}"
+$global:MasterSubnet = "{{GetParameter "masterSubnet"}}" 

```

Comment:
several lines after here, the assignment of global:KubeletNodeLabels  is different from aks-engine as follows because we have labelResourceGroup related change implemented in https://github.com/Azure/AgentBaker/blob/20d879d663566490aeb66f2c10efe7d9281df019/pkg/agent/baker.go#L209 
```
{{if IsKubernetesVersionGe "1.16.0"}}
$global:KubeletNodeLabels = "{{GetAgentKubernetesLabels . "',variables('labelResourceGroup'),'"}}"
{{else}}
$global:KubeletNodeLabels = "{{GetAgentKubernetesLabelsDeprecated . "',variables('labelResourceGroup'),'"}}"
```

Reply:
`"{{GetParameter "masterSubnet"}}" ` brings no harm as it provides subnet of VNetCIDR and it's being used by our current Windows nodepools, but as the first step to enable agentbaker windows, we here keep this for minimal change, and will delete it in the coming PR.